scripts/tests/dfx_tests.sh

```diff
@@ -97,14 +104,19 @@ dfx ledger fabricate-cycles --t 1000000 --canister $wallet_principal
 
 sleep 10
 
+# run tests
+set +e
 cargo test -p integration-tests --features dfx_tests $@
 TEST_RESULT=$?
+mkdir -p .logs/
```

Comment:
can we create the logs folder inside target instead?

Reply:
leftovers because I used to upload logs to github ci to debug the runner. Removed